docs: Consul 1.15.0 and Consul K8s 1.0 release notes (#16481)
hashicorp/consul: website/content/docs/release-notes/consul-k8s/v1_1_x.mdx

---
layout: docs
page_title: 1.1.x
description: >-
  Consul on Kubernetes release notes for version 1.1.x
---

# Consul on Kubernetes 1.1.0

## Release Highlights

> - **Enhanced Envoy Access Logging:** Envoy access logs are now centrally managed via the `accessLogs` field within the ProxyDefaults CRD, to allow operators to easily turn on access logs for all proxies within the service mesh. 

Reviewer: ```suggestion
- **Enhanced Envoy Access Logging:** Envoy access logs are now centrally managed via the `accessLogs` field within the ProxyDefaults CRD to allow operators to easily turn on access logs for all proxies within the service mesh. 
```

> - **Consul Envoy Extensions:** A new extension system has been added to Consul to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase. 

Reviewer: ```suggestion
- **Consul Envoy Extensions:** The new Envoy extension system enables you to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase. 
```

> An `envoyExtensions` field has been added to the ProxyDefaults and ServiceDefaults CRD to enable built-in Envoy extensions. Refer to [Envoy extensions overview](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy-extensions) for more information on how to leverage these extensions.   

Reviewer: ```suggestion
The new `envoyExtensions` field in the ProxyDefaults and ServiceDefaults CRDs enable built-in Envoy extensions. Refer to [Envoy extensions overview](/consul/docs/connect/proxies/envoy-extensions) for more information on how to use these extensions.   
```

## What's Changed

- Connect inject now excludes the `openebs` namespace from sidecar injection by default. If you previously had pods in that namespace
that you wanted to be injected, you must now set namespaceSelector as follows:

  ```yaml
  connectInject:
    namespaceSelector: |
      matchExpressions:
      - key: "kubernetes.io/metadata.name"
        operator: "NotIn"
        values: ["kube-system","local-path-storage"]
  ```
  
## Supported Software

> ~> **Note:** Consul 1.14.x and 1.13.x are not supported. Please refer to [Supported Consul and Kubernetes versions](https://developer.hashicorp.com/consul/docs/k8s/compatibility#supported-consul-and-kubernetes-versions) for more detail on choosing the correct `consul-k8s` version.

Reviewer: ```suggestion
~> **Note:** Consul 1.14.x and 1.13.x are not supported. Please refer to [Supported Consul and Kubernetes versions](/consul/docs/k8s/compatibility#supported-consul-and-kubernetes-versions) for more detail on choosing the correct `consul-k8s` version.
```
